cmd/genji/shell/command.go

```diff
@@ -2,20 +2,21 @@ package shell
 
 import (
 	"fmt"
-	"sort"
-
+	"github.com/agnivade/levenshtein"
 	"github.com/genjidb/genji"
 	"github.com/genjidb/genji/document"
+	"sort"
 )
 
 var commands = map[string]string{
-	".tables":               "\t\tList names of tables.",
-	".exit":                 "\t\t\tExit this program.",
-	".indexes [table_name]": "\tDisplay all indexes or the indexes of the given table name.",
-	".help":                 "\t\t\tList all commands.",
+	".exit":                 "exit this program",
+	".tables":               "shows all tables",
+	".indexes [table_name]": "shows all indexes",
+	".help":                 "shows help",
 }
 
-func runTablesCmd(db *genji.DB, cmd []string) error {
+// displayTables shows all tables.
+func displayTables(db *genji.DB, cmd []string) error {
```

Comment:
I don't think there is a need to rename this function: we are running the tables command, this is not only displaying something.
Think of it this way: everything the user runs that starts with `.` is a command. However, displaying the suggestion is not a command, that's why we named that function `displaySuggestion`.